src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media(min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
 
   &__card {
-    display: flex;
+    display: grid;
+
     justify-content: center;
     align-items: center;
-    width: 200px;
+
     height: 300px;
+
     border: 1px solid #ccc;
```

Comment:
Usually we use vars for colors, sp you can move it to some variable)

Reply:
Don't forget about variables